os_migrate/roles/import_workloads/tasks/workload.yml

```diff
@@ -34,6 +34,51 @@
       dst_filters: "{{ os_migrate_dst_filters }}"
     when: prelim.changed
 
+  # Stop VM if not stopped and if config set to true
+  - name: perform workload stop if os_migrate_workload_stop_before_migration is true
+    openstack.cloud.server_action:
+      auth: "{{ os_migrate_src_auth }}"
+      auth_type: "{{ os_migrate_src_auth_type|default(omit) }}"
+      region_name: "{{ os_migrate_src_region_name|default(omit) }}"
+      server: "{{ item }}"
```

Comment:
`item` is the serialized data structure (complex dict) of the server. We should use the ID here, so `server: "{{ item['_info']['id'] }}"`

Reply:
Thanks - I had seen that in my testing too. Changes made in https://github.com/os-migrate/os-migrate/pull/464/commits/f57d4d2b7c7275ffc061610a38aaef13bdf0c4d6